inscriptions/ISic000760.xml

```diff
@@ -192,14 +197,25 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="unsupplied" xml:lang="la">
-                <ab><!--Greek or Latin (etc.) text here--></ab>
+            <div type="edition" xml:space="preserve" xml:lang="la" resp="#JH">
+                <ab>
+                    <lb n="1"/>Dis Manibus <supplied reason="lost" cert="low">sacrum</supplied>
+                    <lb n="2"/>amicae carissi<supplied reason="lost">mae sepulcrum et mo</supplied>
+                    <lb n="3"/>nimentum fe<supplied reason="lost">ci</supplied> <name><gap reason="lost" extent="unknown" unit="character"/></name>
```

Comment:
`<lb n="3" break="no"/>`
and right idea on the name, but I would suggest:
`<gap reason="lost" extent="unknown" unit="character"><desc>personal name</desc></gap>`
since there is no point in a name tag, since we have no trace of the name to index or otherwise handle.